docs/sdk/pnp/web/adapters/auth.mdx

```diff
@@ -289,23 +282,190 @@ instance. This function takes the below-mentioned parameters as well as
 
 <TabItem value="table">
 
-| Parameter          | type                     |
-| ------------------ | ------------------------ |
-| `clientId?`        | `string`                 |
-| `sessionTime?`     | `number`                 |
-| `chainConfig?`     | `CustomChainConfig`      |
-| `web3AuthNetwork?` | `OPENLOGIN_NETWORK_TYPE` |
+`AuthOptions`
+
+| Parameter               | type                       |
+| ----------------------- | -------------------------- |
+| `clientId`              | `string`                   |
```

Comment:
write description and include type in it

Reply:
@ihsraham can you check?

Reply:
Will add it